BUILD_COMMANDLINE.md

```diff
@@ -1,6 +1,21 @@
 # Building MEI on the command line
 
-1. Check if your system meets the build requirements
+1. Build requirements
+
+   To build all possible artifacts, your build system has to meet the following prerequisites:
+
+   | Prerequisite | Recommended version |
+   |----|----------------|
+   |Java| 17 (at least Java 8 at runtime to support Apache Ant™)|
+   |Apache Ant|1.10.13|
+   |Verovio Toolkit|3.9|
```

Comment:
Why are you recommending 3.9 and not a more recent version ?

Reply:
Just the latest successfully tested at the moment. Haven't tested later releases at the moment. Feel free to test it with a local build of the mei-docker container an file a pull request. That would be great!

Reply:
N.B. pull request both here and in docker-mei; testing at least in docker-mei, If you want, also with in a native command line scenario